### PR TITLE
fix

### PR DESCRIPTION
**Pull request by jonhartzler**

### CODE REVIEW

**Comment by rfox2** [comment]
👍

File: app/controllers/api_controller.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class APIController < ApplicationController
-  protect_from_forgery with: :null_session
+  protect_from_forgery unless: -> { true }
```
Comment (jgondron) [inline app/controllers/api_controller.rb]: What is this logical wizardry?!?!?

**Comment by jgondron** [comment]
👍

**Comment by danthewolfe** [comment]
What's happening here? Where's the description???

**Pull request merged**